zulip_bots/zulip_bots/bots/converter/converter.py

```diff
@@ -83,14 +87,14 @@ def get_bot_converter_response(message, bot_handler):
                     exponent -= exp
                     unit_to = unit_to[len(key):]
 
-            uf_to_std = utils.UNITS.get(unit_from, False)
-            ut_to_std = utils.UNITS.get(unit_to, False)
+            uf_to_std = utils.UNITS.get(unit_from, [])
+            ut_to_std = utils.UNITS.get(unit_to, [])
 
-            if uf_to_std is False:
+            if uf_to_std is []:
```

Comment:
I'm aware that I suggested moving to the empty-list approach here, and you have the right idea I think.
However, your use of `is` here will not work as the objects being compared cannot be the same exact object (this also indicates how we are not testing the converter bot sufficiently, as I think this should cause a test to fail).
You'll need a different approach for the check; let me or a mentor know about this - I know we chatted about the mypy issue before on Zulip.